content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -846,6 +846,48 @@ console.log(checkJson.length)
 </tab>
 </TabbedContent>
 
+## Filtering by Null Values
+
+There are two types of null values possible for a Json field in an SQL database.
+
+- Database NULL: The value in the database is a NULL.
+- JSON null: The value in the database contains a JSON value that is null.
+
+To differentiate between these possibilities, we've introduced three null enums you can use to filter on
+
+- `JsonNull`: Selects the null value in JSON.
+- `DbNull`: Selects the NULL value in the database.
+- `AnyNull`: Selects both null JSON values and NULL database values.
+
+For example,
+
+```prisma
+model Log {
+  id   Int  @id
+  meta Json
+}
+```
+
+```ts
+import { Prisma } from '@prisma/client'
+
+prisma.log.findMany({
+  where: {
+    data: {
+      meta: Prisma.AnyNull,
+    },
+  },
+})
+```
+
+<Admonition type="info">
+
+These null enums does not apply to MongoDB because there is not a difference between a JSON null and a database NULL.
+
+They also do not apply to the `array_contains` operator becuase there can only be a JSON null within an JSON array. Since there cannot be a database NULL within a JSON array, `{ array_contains: null }` is not ambiguous.
```

Comment:
```suggestion
These null enums do not apply to MongoDB because there is not a difference between a JSON null and a database NULL.

They also do not apply to the `array_contains` operator becuase there can only be a JSON null within an JSON array. Since there cannot be a database NULL within a JSON array, `{ array_contains: null }` is not ambiguous.
```